models/sap.yml

```diff
@@ -1,11 +1,975 @@
+      - name: menge
+        description: Quantity
+      - name: pprct
+        description: Partner profit center
+      - name: pfkber
+        description: Functional area of partner
+      - name: wrbtr
+        description: Amount in document currency
+      - name: saknr
+        description: G/L account number
+      - name: xlgclr
+        description: Clearing specific to ledger groups
+      - name: xopvw
+        description: Indicator (Open Item Management?)
+      - name: zzspreg
+        description: Special region
+      - name: xreorg
+        description: Doc. contains open item that Was transferred during reorg.
+      - name: zzbuspartn
+        description: Business partner
+      - name: zzproduct
+        description: Product group
+      - name: zzloca
+        description: City
+      - name: zzchan
+        description: Distribution channel
+      - name: zzlob
+        description: Business line
+      - name: zzuserfld1
+        description: Territory
+      - name: zzuserfld2
+        description: Owner/Cont.
+      - name: zzuserfld3
+        description: Vein
+      - name: zzregion
+        description: User defined regions
+      - name: zzstate 
+        description: State/province code
+
   - name: sap__0gl_account_attr
-    description: Access the general ledger account number attribute.
+    description: G/L Account Number data source
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - mandt
+            - ktopl
+            - saknr
+    columns:
+      - name: mandt
+        description: Client
+      - name: ktopl
+        description: Chart of accounts
+      - name: saknr
+        description: G/L account number
+      - name: bilkt
+        description: Group account number
+      - name: gvtyp
+        description: P&L statement account type
+      - name: vbund
+        description: Company id of tradiing partner
+      - name: xbilk
+        description: Indicator (account is a balance sheet account?)
+      - name: sakan
+        description: G/L account number, significant length
+      - name: erdat
+        description: Date on which the record was created
+      - name: ernam
+        description: Name of person who created the object
+      - name: ktoks
+        description: G/L account group
+      - name: xloev
+        description: Indicator (account marked for deletion?)
+      - name: xspea
+        description: Indicator (account is blocked for creation?)
+      - name: xspeb
+        description: Indicator (Is account blocked for posting?)
+      - name: xspep
+        description: Indicator (Account blocked for planning?)
+      - name: func_area
+        description: Functional area
+      - name: mustr
+        description: Number of the sample account
+
   - name: sap__0material_attr
-    description: Access the material master data from the source system.
\ No newline at end of file
+    description: Material attributes
```

Comment:
Request for more details in description of the model.

Reply:
Updated.